sample_factory/envs/mujoco/mujoco_params.py

```diff
@@ -1,13 +1,80 @@
 def mujoco_override_defaults(env, parser):
     parser.set_defaults(
+        batched_sampling=True,
+        num_workers=8,
+        num_envs_per_worker=16,
+        worker_num_splits=2,
+        train_for_env_steps=1000000,
+
         encoder_type='mlp',
         encoder_subtype='mlp_mujoco',
         hidden_size=64,
         encoder_extra_fc_layers=0,
         env_frameskip=1,
-        nonlinearity='tanh'
+        nonlinearity='tanh',
+        batch_size=1024,
+        # num_batches_per_epoch=10,
+
+        use_rnn=False,
+        adaptive_stddev=False,
+        policy_initialization='torch_default',
+        reward_scale=0.1,
+        rollout=16,
+        max_grad_norm=0.0,
+        ppo_epochs=10,
+        ppo_clip_ratio=0.2,
+        value_loss_coeff=2.0,
+        exploration_loss_coeff=0.0,
+        learning_rate=3e-4,
+        lr_schedule='kl_adaptive_minibatch',
+        lr_schedule_kl_threshold=0.08,
+        shuffle_minibatches=True,
+        gamma=0.99,
+        gae_lambda=0.95,
+        with_vtrace=False,
+        recurrence=1,
+        value_bootstrap=True,  # assuming reward from the last step in the episode can generally be ignored
+        normalize_input=True,
+        experiment_summaries_interval=3,  # experiments are short so we should save summaries often
+        save_every_sec=15,
+
+        serial_mode=False,
```

Comment:
Very cool! Did you get these parameters from baselines or rl-games? Some of them also look similar to isaacgymenvs

Reply:
RL games and baselines didn't provide many parameters (mostly just LR, batch size and gamma values) so the rest I copied from isaac gym and tuned from there.